UnityProject/Assets/Scripts/Managers/SubSceneManager/SubSceneManager.Server.cs

```diff
@@ -88,14 +92,16 @@ IEnumerator AddObserversForClient(NetworkConnectionToClient connToAdd, Scene sce
 				yield break;
 
 			n.AddPlayerObserver(connToAdd);
-			objCount++;
-			if (objCount >= 20)
+
+			if (Stopwatch.ElapsedMilliseconds >= 10)
```

Comment:
how does this work

Reply:
if it takes too long then skips for a frame